src/modules/delete-task.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/no-cycle */
+import { todoList, saveData } from './add-task.js';
+import { renderList } from './todolist.js';
+
+function deleteTask(event) {
+  const { index } = event.target.dataset;
+  todoList.splice(index, 1);
+  saveData();
+
+  // update the index value of the remaining tasks
+  for (let i = index; i < todoList.length; i += 1) {
+    todoList[i].index = i;
+  }
+
+  renderList();
+}
```

Comment:
- [ ] As requested by the previous code reviewer, the index should start in one and update when it removes some item from the array, at the moment of deleting the first element from the array with index 0 it returned an error, you can see it below. Please 🙏🏽 correct the indexes to avoid errors in removing items.

![image](https://user-images.githubusercontent.com/58642949/230675107-5fd14d1d-8a0e-433b-84cc-79acb8253786.png)

![image](https://user-images.githubusercontent.com/58642949/230675162-6260c719-d0e3-446d-9b0e-2223fbd6b198.png)